NetworkManager-0.8 renamed libnm_glib to libnm-glib

http://bugs.gentoo.org/show_bug.cgi?id=306977

Patch found by Andrea Pinazzi on the debian bugzilla
---
--- configure.ac
+++ configure.ac
@@ -40,7 +40,7 @@
 GEOCLUE_REQUIRED=0.11
 WEBKIT_REQUIRED=1.1.7
 KEYRING_REQUIRED=2.22
-NETWORK_MANAGER_REQUIRED=0.7.0
+NETWORK_MANAGER_REQUIRED=0.8
 
 # Use --enable-maintainer-mode to disabled deprecated symbols
 GNOME_MAINTAINER_MODE_DEFINES
@@ -184,7 +184,7 @@
 
    PKG_CHECK_MODULES(NETWORK_MANAGER,
    [
-      libnm_glib >= $NETWORK_MANAGER_REQUIRED
+      libnm-glib >= $NETWORK_MANAGER_REQUIRED
    ], have_nm="yes", have_nm="no")
 
    if test "x$have_nm" = "xyes"; then
